Source/WTF/wtf/text/CString.cpp

```diff
@@ -111,7 +111,7 @@ void CString::grow(size_t newLength)
     ASSERT(newLength > length());
 
     auto newBuffer = CStringBuffer::createUninitialized(newLength);
-    memcpy(newBuffer->mutableData(), m_buffer->data(), length() + 1);
```

Comment:
This would need to be strncpy to avoid introducing new attack surface. That being said, what benefit did you observe doing this?